rcp/org.testeditor.rcp4.uatests/src/main/java/org/testeditor/fixture/swt/SWTFixture.java

```diff
@@ -398,14 +417,44 @@ public void executeContextMenuEntry(String menuItem, String viewName, SWTLocator
 		try {
 			item = ContextMenuHelper.contextMenu(tree, menuItem.split("/"));
 		} catch (WidgetNotFoundException e) {
-			Thread.sleep(100);
-			logger.trace("Search menu entry a second time: {}", menuItem);
-			item = ContextMenuHelper.contextMenu(tree, menuItem.split("/"));
+			logger.error("exception during executeContextMenuEntry", e);
+			dumpMenuStatusFor(menuItem, viewName, locatorStrategy);
 		}
 		assertNotNull(item);
 		logger.trace("Click on menu item: {}", menuItem);
 		new SWTBotMenu(item).click();
 	}
+	
+	private void dumpMenuStatusFor(String menuItem, String viewName, ViewLocatorStrategy locatorStrategy) {
+		getDisplay().syncExec(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					logger.debug("Search for tree in view = '{}' with strategy = '{}'", viewName,
+							locatorStrategy.name());
+					String[] menuItems = menuItem.split("/");
```

Comment:
The naming is confusing here. `menuItem` is actually `menuItemPath` and the split result `pathElements`